### PR TITLE
Cred handling update

### PR DESCRIPTION
**Pull request by binaryaaron**
This would address #7 , cleaning up some of the credential management.

This simplifies credential management to this:

```.python
search_args, count_args = load_credentials("~/.twitter_keys.yaml",
                                                                          account_type="enterprise")
```

I have not worked on improving it from the command-line utility just yet.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -145,35 +146,46 @@ Working with the API within a Python program is straightforward both for
...
-credentials is valid here; I am using environment variables for ease of
-use.
+throughout your program's session.
 
 .. code:: python
 
     import os
     import json
-    from twittersearch import ResultStream, gen_rule_payload
+    from unidecode import unidecode
```
Comment (jrmontag) [inline README.rst]: Might be helpful to mention that the unidecode library isn't necessary, but (I assume) is helpful for display in the context of this readme. At least that's what it looks like to me.

Comment (binaryaaron) [reply README.rst]: I removed the unidecode stuff ; I can strip out any unicode from the readme if needed. I agree that it adds some confusion or something in there.

**Comment by binaryaaron** [comment]
@jrmontag - I added the changes we discussed. 

If the need arises, we can add a flag for the `counts` endpoint to be used elsewhere, but I do like the single point of access and think all of these changes do simplify the usage a good deal.

File: twittersearch/api_utils.py [hunk 2]
```diff
@@ -251,6 +263,4 @@ def load_credentials(filename=None, account_type=None):
         search_args = {"username": search_creds["username"],
                        "password": search_creds["password"],
                        "endpoint": search_creds["endpoint"]}
-    count_args = {**search_args,
-                  **{"endpoint": change_to_count_endpoint(search_args["endpoint"])}}
-    return search_args, count_args
+    return search_args
```
Comment (jrmontag) [inline twittersearch/api_utils.py]: Good simplification of the return type here. 

Based on this new version, I think the one additional thing I'd like to see in this method is user-friendly exception handling for the KeyErrors that will come from a misconfigured yaml file. The simplest version might be a try/except (just for `KeyError`) around the two conditional statements with a helpful error message along the lines of "ensure that your credentials file has the proper keys and values for the endpoint you're trying to use. see the readme for examples."

**Comment by binaryaaron** [comment]
Added your last suggestion, @jrmontag, and will merge this into master. We'll need to update the command-line app soon, but I'll leave that as a WIP for now.

**Pull request merged**